lib/nmatrix/numruby.rb

```diff
@@ -62,4 +62,27 @@ def self.hstack(objs)
     result
   end
 
+  def self.tri(N, M=None, k=0, nm_dtype=float)
+    if M is None:
```

Comment:
I suppose that `None` is an object that you created. But I think that you can use `nil` instead of it. Also I think that isn't necessary use the `:` to the end of sentence. WDYT? 🤔

Reply:
I think `nil` will work fine also I have removed `:` from code because I was referring to `numpy` library of Python but I have gone through previous Ruby syntax of `numruby` and I think there is no need of `:` after `nil` @developerfab thank you for the suggestion